lib/screens/details_page/details.dart

```diff
@@ -32,267 +30,315 @@ class Details extends StatefulWidget {
 class _DetailsState extends State<Details> {
   @override
   Widget build(BuildContext context) {
-  String price = widget.price;
+    var additions = ['blue cheese', 'mushroom'];
+    String val = 'blue cheese';
+    String price = widget.price;
```

Comment:
@batsy3 are we not getting data from the api? Do we need these static values here?

Reply:
im doing these in a seperate pr